### PR TITLE
Ability to reload Oxidized node list, when a device is added

### PR DESCRIPTION
**Pull request by Rosiak**

### CODE REVIEW

File: includes/defaults.inc.php [hunk 1]
```diff
@@ -473,6 +473,7 @@ function set_debug($debug) {
 $config['smokeping']['pings']             = 20;
 // $config['oxidized']['enabled']         = FALSE;//Set to TRUE
 // $config['oxidized']['url']             = 'http://127.0.0.1:8888';// Set the Oxidized rest URL
+// $config['oxidized']['reload_nodes'] = FALSE;//Set to TRUE, check documentation
```
Comment (laf) [inline includes/defaults.inc.php]: This should be uncommented out.

File: includes/functions.php [hunk 2]
```diff
@@ -580,7 +580,10 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         if (host_exists($host) === false) {
             $device_id = dbInsert($device, 'devices');
             if ($device_id) {
-                return($device_id);
+                if ($config['oxidized']['enabled'] === TRUE && $config['oxidized']['reload_nodes'] === TRUE && isset($config['oxidized']['url'])) {
+                    $oxidized_reload_response = oxidized_reload_nodes();
+                }
+                return($device_id . $oxidized_reload_response);
```
Comment (laf) [inline includes/functions.php]: You can't return more than device_id here as it's used when calling the function.

File: includes/functions.php [hunk 3]
```diff
@@ -580,7 +580,10 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         if (host_exists($host) === false) {
             $device_id = dbInsert($device, 'devices');
             if ($device_id) {
-                return($device_id);
+                if ($config['oxidized']['enabled'] === TRUE && $config['oxidized']['reload_nodes'] === TRUE && isset($config['oxidized']['url'])) {
```
Comment (laf) [inline includes/functions.php]: Worth moving this if check line into the function and just call the function. That way if we reuse the function we don't need to duplicate the if check line.

**Comment by Rosiak** [comment]
Closing due to my git mess.

**Pull request closed without merging**